spec/factories/clients.rb

```diff
@@ -94,7 +100,7 @@
     end
 
     factory :client_with_intake_and_return do
-      with_return
+      with_ctc_return
```

Comment:
since there's also `client_with_ctc_intake_and_return` it seems like this one either maybe doesn't need to exist or should it possibly be for gyr?